test/e2e/runner/load.go

```diff
@@ -95,32 +96,56 @@ func loadGenerate(ctx context.Context, chTx chan<- types.Tx, multiplier int, siz
 func loadProcess(ctx context.Context, testnet *e2e.Testnet, chTx <-chan types.Tx, chSuccess chan<- types.Tx) {
 	// Each worker gets its own client to each node, which allows for some
 	// concurrency while still bounding it.
-	clients := map[string]*rpchttp.HTTP{}
+
+	clients := make([]*rpchttp.HTTP, 0, len(testnet.Nodes))
+
+	for idx := range testnet.Nodes {
+		if testnet.Nodes[idx].Mode == e2e.ModeSeed {
+			continue
+		}
+		client, err := testnet.Nodes[idx].Client()
+		if err != nil {
+			continue
+		}
+
+		clients = append(clients, client)
+	}
+
+	if len(clients) == 0 {
+		panic("no clients to process load")
+	}
+
+	clientRing := ring.New(len(clients))
```

Comment:
```suggestion
	// Put the clients into a ring so they can be used in round-robin fashion
	clientRing := ring.New(len(clients))
```

or whatever is true here. I think that's what the code below is doing, anyway.

Reply:
nit: seems like we could just create the ring on line `100` since we know the length there and then append to the list in the loop.